servant-server/src/Servant/Server/Internal/RoutingApplication.hs

```diff
@@ -246,7 +266,8 @@ runDelayed Delayed{..} env = runDelayedIO $ do
   methodD
   a <- authD
   b <- bodyD
-  DelayedIO (\ req -> return $ serverD c a b req)
+  p <- paramsD -- Has to be after body to respect the relative error order
```

Comment:
Here it seems like params are being checked after the body. But in the documentation on line 91 it states that params come first. Which one is it?